pingcap/ticdc
# owner: check if the checkpoint ts is less then gc safe time when create the changefeed

### What problem does this PR solve? 
check if the checkpoint ts is less then gc safe time when create the changefeed
fix #1014 

#### Blocked By tikv/pd#3128 tikv/pd#3135 tikv/pd#3136 tikv/pd#3173

### Check List 

Tests 

 - Integration test
the `changefeed_error` test in the current integration test is already do good.
 - Manual test (add detailed scripts or steps below)
try to follow the step in #1014, we can see the owner throw an error immediately for now.

### Release note



- No release note
